### DISCUSSION TITLE
Whether to implement RTSP authentication.

### DISCUSSION THREAD

**Discussion by blu310** [SuRGeoNix/Flyleaf#495] [Q&A] [answered]
Hello.
I am using flyleaf well. It's so good.
I have one question.
Is there any implementation of RTSP authentication (digest)?
I need to authenticate by putting ID and PW.
I'm using ID:PW@IP in the Url, but it's a security risk to do it this way.
I would be grateful if you could let me know.
Thank you

**Comment by blu310** [comment]
Checking the actual packet, we see that the ID PW is not exposed.
I think this is an unnecessary question. 😢

**Comment by SuRGeoNix** [comment, marked as answer]
No problem @blu310, this will be hashed-encrypted by the protocol. Let me know if you want me to look further into it.
